src/lib/core/say.c

```diff
@@ -857,15 +857,12 @@ say_format_plain(struct log *log, char *buf, int len, int level,
 	if (module != NULL)
 		SNPRINT(total, snprintf, buf, len, "/%s", module);
 
-	if (level == S_WARN || level == S_ERROR || level == S_SYSERROR) {
-		/* Primitive basename(filename) */
-		if (filename) {
-			for (const char *f = filename; *f; f++)
-				if (*f == '/' && *(f + 1) != '\0')
-					filename = f + 1;
-			SNPRINT(total, snprintf, buf, len, " %s:%i", filename,
-				line);
-		}
+	/* Primitive basename(filename) */
+	if (filename != NULL) {
+		for (const char *f = filename; *f; f++)
+			if (*f == '/' && *(f + 1) != '\0')
+				filename = f + 1;
+		SNPRINT(total, snprintf, buf, len, " %s:%i", filename, line);
```

Comment:
Is it OK in regards to the syslog message format?

Also, I would reflect it in the comment at top of the function.

Reply:
> Is it OK in regards to the syslog message format?

Don't see anything prohibiting this in [RFC 3164](https://datatracker.ietf.org/doc/html/rfc3164#section-4.1.3).

> Also, I would reflect it in the comment at top of the function.

Done.

Reply:
I was under a false impression that this part of the log entry is structured according to some common syslog format. It seems, it is just a free-form MSG part. OK so.